example/full/instance.properties

```diff
@@ -378,6 +378,85 @@ sleeper.bulk.import.emr.ebs.volume.type=gp2
 
+# The architecture for EMR Serverless to use. X86_64 or ARM (Coming soon)
+sleeper.bulk.import.emr.serverless.architecture=X86_64
+
+# The version of EMR Serverless to use.
+sleeper.bulk.import.emr.serverless.release=emr-6.10.0
+
+# The name of the repository for the EMR serverless container. The Docker image from the bulk-import
+# module should have been uploaded to an ECR repository of this name in this account.
+sleeper.bulk.import.emr.serverless.repo=<insert-unique-sleeper-id>/bulk-import-runner-emr-serverless
+
+# The number of cores used by an Serverless executor. Used to set spark.executor.cores.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.executor.cores=4
+
+# The amount of memory allocated to a Serverless executor. Used to set spark.executor.memory.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.executor.memory=16g
+
+# The amount of storage allocated to a Serverless executor.
+sleeper.bulk.import.emr.serverless.executor.disk=200g
+
+# The number of executors to be used with Serverless. Used to set spark.executor.instances.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.executor.instances=36
+
+# The number of cores used by the Serverless Spark driver. Used to set spark.driver.cores.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.driver.cores=4
+
+# The amount of memory allocated to the Serverless Spark driver. Used to set spark.driver.memory.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.driver.memory=16g
+
+# The path to JAVA_HOME to be used by the custom image for bulk import.
+sleeper.bulk.import.emr.serverless.java.home=/usr/lib/jvm/jre-11
+
+# Whether Spark should use dynamic allocation to scale resources up and down. Used to set
+# spark.dynamicAllocation.enabled. See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.dynamic.allocation.enabled=false
+
+# Whether to compress serialized RDD partitions. Used to set spark.rdd.compress.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.spark.rdd.compress=true
+
+# Whether to compress map output files. Used to set spark.shuffle.compress.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.spark.shuffle.compress=true
+
+# Whether to compress data spilled during shuffles. Used to set spark.shuffle.spill.compress.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.spark.shuffle.spill.compress=true
+
+# The default parallelism for Spark job. Used to set spark.default.parallelism.
+# See https://spark.apache.org/docs/latest/configuration.html.
+sleeper.bulk.import.emr.serverless.spark.default.parallelism=290
```

Comment:
I think this should be 2 * 36 * 4 = 288.

Reply:
Updated to 288